web-console/3-project-details/labels.md

```diff
@@ -0,0 +1,99 @@
+# Labels on Detail Pages
+
+## View Labels
+- Labels are currently displayed at the top of most detail pages (Deployments, Pods, Builds, Images, Storage, etc)
+- A new `Labels` section should be added to the bottom of the `Details` tab on all of these pages to allow for a more comprehensive labels area
+
+![Label 1](img/new-labels-00-view.png)
+
+## Add Labels
+![Label 1](img/new-labels-01-add.png)
+
+- A `Manage Labels` link should be displayed below the applied labels
+- Clicking the link will open a modal, displaying the labels that are already applied, as well as an option to add new labels.
+- Clicking `Add` will display a new row with two empty fields for label Name and label Value.
+
+![Label 1](img/new-labels-02-add.png)
+
+- Both empty fields should use the combobox component as users can either select an existing label or enter a new one.
```

Comment:
What does it look like if I type an invalid value? (Labels can only have certain characters.)

Reply:
I'll add an image of that error 👍